src/vs/workbench/contrib/inlineChat/browser/inlineChatController.ts

```diff
@@ -615,6 +623,10 @@ export class InlineChatController implements IEditorContribution {
 			this._zone.value.widget.updateStatus('');
 			this._zone.value.widget.updateMarkdownMessage(renderedMarkdown.element);
 			this._zone.value.widget.updateToolbar(true);
+			const content = renderedMarkdown.element?.textContent;
```

Comment:
`renderedMarkdown.element` should never be undefined or null. Did you experience something different or why the `?` here?